kotlin-ktor/src/test/java/com/baeldung/graphql/server/AttendeeTest.kt

```diff
@@ -0,0 +1,76 @@
+package com.baeldung.graphql.server
+
+import com.baeldung.graphql.graphQlTestEnvironment
+import com.baeldung.graphql.serializeQuery
+import com.baeldung.graphql.server.data.Attendee
+import com.baeldung.graphql.server.data.AttendeeRepository
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import junit.framework.TestCase
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertNotNull
+
+private const val CREATE_ATTENDEE_QUERY = """
+    mutation {
+        saveOrCreateAttendee(attendee: { name: "John Johnson" }) {
+            id
+            name
+      }
+    }
+    """
+
+private const val UPDATE_ATTENDEE_QUERY = """
+    mutation {
+        saveOrCreateAttendee(attendee: { id: 0, name: "Jake Jakeson" }) {
+            id
+            name
+      }
+    }
+    """
+
+class AttendeeTest {
```

Comment:
Can we rename this to `GraphQLServerAttendeeTest` or similar, to emphasise we are testing the server side in this test?

Reply:
Since all the other mutations/queries are covered, should we also include an `objectById` test here too for completeness?

Reply:
Add the TestCase import

Reply:
1. Since this is in the server package, I'm not sure this really makes sense?
2. ✅
3. Not sure what you mean, the import is there for me?

Reply:
1. I think I meant to be consistent with the `GraphQLClientTest` naming, which is in the client package
3. I meant, add the static import so that `TestCase.assertEquals` becomes just `assertEquals`